python-django/README.md

```diff
@@ -2,25 +2,21 @@
 
 A sample application for debugging Django using Rookout.
 
-Before following this guide we recommend reading the basic [Python + Rookout] guide.
-
-This sample may be out of date. If you face any issues, please reach out to mailto:support@rookout.com and let us know.
-
-* [Running Django Server with Rookout](#running-django-server-with-rookout)
-* [Rookout integration explained](#Rookout-integration-explained)
+Before following this guide we recommend reading the basic [Python + Rookout](https://docs.rookout.com/docs/rooks-setup.html) guide.
 
 ## Running Django Server with Rookout
-1. *Run the Rookout agent*:
-``` bash
-$ docker run -p 7486:7486 -e "ROOKOUT_TOKEN=<Your-Token>" rookout/agent
-```
 
-2. *Install dependencies*:
+1. *Install dependencies*:
```

Comment:
maybe add a step before for cloning the repo - look at other up-to-date examples we have and have the readme be similar

Reply:
or merge that step to be Clone and install dependencies

Reply:
Done

Reply:
👍